log-collector-script/linux/eks-log-collector.sh

```diff
@@ -800,8 +801,14 @@ get_io_throttled_processes() {
   ok
 }
 
+get_reboot_history() {
+  try "collect reboot history"
+  timeout 75 last reboot > "${COLLECT_DIR}"/system/last_reboot.txt 2>&1 || echo -e "\tTimed out, ignoring \"reboot history output \" "
+  ok
+}
+
 get_nvidia_bug_report() {
-  try "Collect Nvidia Bug report"
+  try "collect Nvidia Bug report"
```

Comment:
Can you back out these changes? Lets keep the git history clean if there's no logical change

Reply:
Sure. :+1:

Reply:
Done.